receiver/k8sclusterreceiver/internal/node/testdata/expected_optional.yaml

```diff
@@ -22,9 +22,12 @@ resourceMetrics:
         - key: os.description
           value:
             stringValue: Ubuntu 22.04.1 LTS
-        - key: os.version
+        - key: os.kernel.version
```

Comment:
lgtm, but how about we [add it to the semantics-convention](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/os.md?plain=1#L12) and see if the group has any recommendations?